lsp-mode.el

```diff
@@ -841,7 +841,8 @@ Changes take effect only when a new session is started."
                                         (magik-mode . "magik")
                                         (idris-mode . "idris")
                                         (idris2-mode . "idris2")
-                                        (gleam-mode . "gleam"))
+                                        (gleam-mode . "gleam")
+                                        (markdown-mode . "marksman"))
```

Comment:
This change is not needed - markdown-mode is mapped to `markdown`.

Reply:
I was wondering about that.  Thanks.